### PR TITLE
Edit text branch

### PR DESCRIPTION
**Pull request by ashleyrchen**
implemented toggle button so you can turn bugshaker on/off in settings

### CODE REVIEW

**Comment by TRIPA013** [comment]
This currently has 2 commits. I think we only need the second one?

File: bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java [hunk 1]
```diff
@@ -216,8 +219,28 @@ public void start() {
...
 		logger.d("Shake detected!");
-		feedbackEmailFlowManager.startFlowIfNeeded(application,
-			false);
+		//if (bugShakerOn){
+			feedbackEmailFlowManager.startFlowIfNeeded(application,
+				false);
+		//}
+	}
+
+	public static void changeStatus(){
```
Comment (TRIPA013) [inline bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java]: are we using this anywhere?

File: bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java [hunk 2]
```diff
@@ -216,8 +219,28 @@ public void start() {
 	@Override
 	public void hearShake() {
 		logger.d("Shake detected!");
-		feedbackEmailFlowManager.startFlowIfNeeded(application,
-			false);
+		//if (bugShakerOn){
```
Comment (TRIPA013) [inline bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java]: remove commented out code

File: bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java [hunk 3]
```diff
@@ -216,8 +219,28 @@ public void start() {
...
+			feedbackEmailFlowManager.startFlowIfNeeded(application,
+				false);
+		//}
```
Comment (TRIPA013) [inline bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java]: remove commented out code

File: bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java [hunk 4]
```diff
@@ -50,6 +50,8 @@
 	// Instance configuration state:
 	private boolean assembled = false;
 	private boolean startAttempted = false;
+	private static boolean bugShakerOn = false;
```
Comment (TRIPA013) [inline bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java]: do we need this?

File: bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java [hunk 5]
```diff
@@ -216,8 +219,28 @@ public void start() {
...
+	public static void changeStatus(){
+		if (bugShakerOn){
+			bugShakerOn= false;
+		}
+		else if (!(bugShakerOn)){
+			bugShakerOn = true;
+		}
 	}
 
+	public static boolean isBugShakerOn(){
```
Comment (TRIPA013) [inline bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java]: do we need this?

File: bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java [hunk 6]
```diff
@@ -216,8 +219,28 @@ public void start() {
...
+		return bugShakerOn;
+	}
+
+	public static void turnOff(){
+		shakeDetector.stop();
```
Comment (TRIPA013) [inline bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java]: 👍

File: bugshaker/src/main/java/com/github/stkent/bugshaker/MainActivity.java [hunk 7]
```diff
@@ -55,12 +58,36 @@ protected void onCreate(Bundle savedInstanceState) {
...
 		findViewById(R.id.sendEmail).setOnClickListener(this);
-		findViewById(R.id.speechBox).setOnClickListener(this);
+		findViewById(R.id.textEdit).setOnClickListener(this);
+
+		annotatedTextBox = (EditText) findViewById(R.id.textEditing);
+		annotatedTextBox.setOnTouchListener(new View.OnTouchListener() {
+			@Override
+			public boolean onTouch(View view, MotionEvent event) {
+
+				//TODO: figure out if this works or not!!!!!!
```
Comment (TRIPA013) [inline bugshaker/src/main/java/com/github/stkent/bugshaker/MainActivity.java]: we can remove this comment!

File: bugshaker/src/main/java/com/github/stkent/bugshaker/MainActivity.java [hunk 8]
```diff
@@ -128,6 +155,9 @@ else if (view.getId() == R.id.erase_btn) {
 		else if (view.getId() == R.id.sendEmail) {
 			saveSendScreenshotAndLog();
 		}
+		else if(view.getId()==R.id.textEdit){
```
Comment (TRIPA013) [inline bugshaker/src/main/java/com/github/stkent/bugshaker/MainActivity.java]: we can format this line.

File: bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java (hunk 4 above)
Comment (ashleyrchen) [inline bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java]: we need this for isBugShakerOn which is called by EBPreferencesFragment!

File: bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java (hunk 5 above)
Comment (ashleyrchen) [inline bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java]: EBPreferencesFragment calls this to check whether bugshaker is already listening to shakes at that moment

**Comment by TRIPA013** [comment]
Good work! 👍

**Pull request merged**